src/tileanimation.cpp

```diff
@@ -19,23 +19,8 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include "tileanimation.h"
 #include "util/serialize.h"
 
-void TileAnimationParams::serialize(std::ostream &os, u16 protocol_version) const
+void TileAnimationParams::serialize(std::ostream &os, u8 version) const
```

Comment:
Why 'version' as an argument?

Reply:
this makes perfect sense for the `serialize` function, but not for the `deserialize` one.

Reply:
yeah it should be removed, also protocol version is a 16b length version since protocol v27 (can't remember if it's the exact version)

Reply:
As before, it will likely be used in future, i'm fine with it present or removed, but maybe 'protocol_version' is better because 'version' suggests a smaller number as elsewhere?